assets/app/views/project.html

```diff
@@ -42,173 +55,152 @@ <h2 class="route osc-object" ng-if="displayRouteByService[serviceId]" ng-init="o
-                    <span ng-if="deployment.details && deployment.details.causes && deployment.details.causes.length > 0">
-                      <span>, triggered by 
-                        <span class="deployment-trigger" ng-repeat="cause in deployment.details.causes">
-                          <span ng-switch="cause.type">
-                            <span ng-switch-when="ImageChange">new image for {{cause.imageTrigger.repositoryName | imageStreamName}}:{{cause.imageTrigger.tag}}</span>
-                            <span ng-switch-when="ConfigChange">deployment configuration change</span>
-                          </span>
-                        </span>
-                      </span>
-                    </span>
-                  </div>
-                  <div class="pod-block">
-                    <!-- TODO figure out why podTemplate can't be done the same way as pods -->
-                    <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
-                    <pods pods="podsByDeployment[deployment.metadata.name]"></pods>
-                  </div>
-                </div>
-              </div>
-            </div>
-            <!-- Now go through any of the pods that are routed to by the service but not part of a deployment -->
-              <div ng-repeat="(podName, pod) in monopodsByService[serviceId]">
-                <div class="ng-scope tile-container">
-                  <div class="metadata">
-                    <relative-timestamp timestamp="pod.metadata.creationTimestamp"></relative-timestamp>
-                  </div>                
-                  <div class="pod-block">
-                    <pod-template ng-init="podTemplate = pod"></pod-template>
-                    <pods pods="[pod]"></pods>
-                  </div>
-                </div>
-              </div>
-              <!-- TODO implement filters for empty and present to return booleans for cases like this -->
-            <div ng-if="(podsByService[serviceId] | hashSize) == 0" class="osc-object tile-container deployments">
-              <span style="margin-left: 20px;">There are currently no pods for this service.</span>
             </div>
+
+            <div class="component meta-data">
+              <span ng-if="service.spec.portalIP != 'None'">
+                routing traffic on {{service.spec.portalIP}}
+              </span>
+              <span ng-if="numPorts"> -
+                <!--
+                Show only the first two ports if there are many since we don't have much space here.
+                The full list is visible elsewhere.
+                -->
+                <span ng-repeat="portMapping in service.spec.ports | orderBy:port | limitTo:2">
+                  port {{portMapping.port}} &#8594; {{portMapping.targetPort}}
+                    ({{portMapping.protocol}})<span ng-if="$index < (numPorts - 1)">, </span>
```

Comment:
Probably better to set it to be left aligned beneath the name when width < 400px, as such
![blocked](https://cloud.githubusercontent.com/assets/1874151/7727718/b321af18-fed8-11e4-87e7-902fd409d806.png)